tests_zemu/package.json

```diff
@@ -6,8 +6,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest -f ./tests/test.js --detectOpenHandles",
-    "test_val": "jest -f ./tests/val.test.js --detectOpenHandles"
+    "test": "jest --detectOpenHandles -t 'Standard'",
+    "testval": "jest --detectOpenHandles -t 'Validator'"
```

Comment:
I would not recommend using the `-t` argument. It is useful during dev when working on fixing one test but in the CI it then count others tests as skipped. Also we can't then used the `-t` argument as it will be ignored and the first will be kept.